gatsby-browser.js

```diff
@@ -1,5 +1,10 @@
-exports.shouldUpdateScroll = ({ prevRouterProps, routerProps }) => {
-    const prevLocation = prevRouterProps && prevRouterProps.location.pathname;
-    const newLocation = routerProps && routerProps.location.pathname;
+import { removeTrailingSlash } from './src/utils/remove-trailing-slash';
+
+export const shouldUpdateScroll = ({ prevRouterProps, routerProps }) => {
```

Comment:
I noticed sometimes the page would jump to the top when a filter was applied. That was because sometimes there is a trailing slash initially which then gets removed when the filters are applied. This check now accounts for any potential trailing slashes